warehouse/localdisk.go

```diff
@@ -0,0 +1,130 @@
+package warehouse
+
+import (
+	"context"
+	"fmt"
+	"github.com/fullstorydev/hauser/config"
+	"github.com/nishanths/fullstory"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"time"
+)
+
+type LocalDisk struct {
+	conf *config.Config
+	ctx  context.Context
+}
+
+const (
+	timestampFile string = ".sync.hs"
+)
+
+var _ Warehouse = (*LocalDisk)(nil)
+
+func NewLocalDisk(c *config.Config) *LocalDisk {
+	if c.Local.UseStartTime {
+		filename := filepath.Join(c.Local.SaveDir, timestampFile)
+		if _, err := os.Stat(filename); !os.IsNotExist(err) {
+			os.Remove(filename)
+		}
+	}
+
+	return &LocalDisk{
+		conf: c,
+	}
+}
+
+// Currently returns the names of all the fields exported by BundleFields
+func (w *LocalDisk) GetExportTableColumns() []string {
+	allFields := BundleFields()
+	fieldNames := make([]string, 0, len(allFields))
+	for name, _ := range allFields {
+		fieldNames = append(fieldNames, name)
+	}
+	return fieldNames
+}
+
+// Reads the last sync date from file
+func (w *LocalDisk) LastSyncPoint() (time.Time, error) {
+	t := beginningOfTime
+	if w.conf.Local.UseStartTime {
+		t = w.conf.Local.StartTime
+	}
+	if _, err := os.Stat(w.conf.Local.SaveDir); os.IsNotExist(err) {
+		panic(err)
+	}
+	filename := filepath.Join(w.conf.Local.SaveDir, timestampFile)
+	if _, err := os.Stat(filename); os.IsNotExist(err) {
+		return t, nil
+	}
+	timebytes, err := ioutil.ReadFile(filename)
+	if err != nil {
+		return t, err
+	}
+	synctime, err := time.Parse(time.RFC3339, string(timebytes))
+	return synctime, err
+}
+
+// Writes the current time to timestamp file
+func (w *LocalDisk) SaveSyncPoints(bundles ...fullstory.ExportMeta) error {
+	t := time.Now().UTC().Format(time.RFC3339)
+	if _, err := os.Stat(w.conf.Local.SaveDir); os.IsNotExist(err) {
+		panic(err)
+	}
+	filename := filepath.Join(w.conf.Local.SaveDir, timestampFile)
+	timedata := []byte(t)
+	err := ioutil.WriteFile(filename, timedata, 0644)
```

Comment:
return ioutil.WriteFile(filename, timedata, 0644) directly since the err isn't used in this scope

Reply:
Will do